packages/babylon/src/types.js

```diff
@@ -631,7 +634,7 @@ export type ClassMemberBase = NodeBase &
     // TypeScript only:
     accessibility?: ?Accessibility,
     abstract?: ?true,
-    optional?: ?true,
+    optional?: ?true, //!
```

Comment:
Did you forgot `exclamation`?

Reply:
Actually that properly goes on `ClassProperty` only, not on any other class members. Left a comment behind though.